manifests/infrastructure.pp

```diff
@@ -20,9 +20,9 @@
   include wifi
 
   if 64 == $psk.length() {
```

Comment:
This condition looks odd to me! How to set a PSK 64 characters long but not hashed?

Reply:
I was following the `wpa_supplicant.conf` man page which says:

> The key is specified as 64 hex digits or as an 8-63 character ASCII passphrase.

[This stackoverflow answer](https://stackoverflow.com/questions/18006390/why-is-the-wpa2-psk-key-length-limited-to-63-characters) quote *802.11i specification* which is consistent:

> pass-phrase is a sequence of between 8 and 63 ASCII-encoded characters. The limit of 63 comes from the desire to distinguish between a pass-phrase and a PSK displayed as 64 hexadecimal characters.

Reply:
Ouch! OK, thanks!